text/0006-monolothic-packaging.md

```diff
@@ -144,11 +146,13 @@ The current thinking is to *statically link* the protocol definitions between th
 - [ ] Rewrite all example code (see [cdk-rewrite-mono-import](https://github.com/rix0rrr/cdk-rewrite-mono-imports)).
 - [ ] Reference documentation needs to also support submodules/namespaces and use the submodule's README file.
 
+
+
 # Drawbacks
 
 - [ ] In JavaScript/TypeScript code will always be fully qualified
   (`aws_s3.Bucket` instead of `Bucket`) due to limitations of the `import` tool.
-- [ ] The size of the single module will be large (~50MiB) which can create issues of using the CDK in non-standard environments such as from a Lambda function.
+- [ ] The size of the single module will be large (~50MiB) which can create issues of using the CDK in non-standard environments such as from a Lambda function. See this https://github.com/aws/aws-cdk-rfcs/issues/39#issuecomment-593092612
```

Comment:
You could add that if needed, we could ship a mini ford version of the MonoCDK, which would cut down the size significantly (see [here](https://bundlephobia.com/result?p=monocdk-experiment@1.26.0) for reference)

Reply:
added